plugins/network-elements/nuage-vsp/src/net/nuage/vsp/acs/client/NuageVspSyncClient.java

```diff
@@ -21,7 +21,7 @@
 
 public interface NuageVspSyncClient {
 
-    public void syncWithNuageVsp(String nuageVspEntity) throws Exception;
+    void syncWithNuageVsp(String nuageVspEntity) throws Exception;
```

Comment:
This is how the methods are declared in our client. They need to have the same signature as defined in our client.